plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/ProjectFromSelectionHelper.java

```diff
@@ -32,7 +32,7 @@ public IProject getProject(ExecutionEvent event) throws CoreException, Execution
 
         IFacetedProject facetedProject = facetedProjectHelper.getFacetedProject(project);
         // TODO replace with constant from AppEngineFacet (after possibly relocating that class)
-        if (facetedProjectHelper.projectHasFacet(facetedProject, "com.google.cloud.tools.eclipse.appengine.facet")) {
+        if (facetedProjectHelper.projectHasFacet(facetedProject, "com.google.cloud.tools.eclipse.appengine.facets.standard")) {
```

Comment:
Also we should use `com.google.cloud.tools.eclipse.appengine.facets.AppEngineStandardFacet.ID` instead of the string literal (as the TODO says)